src/redux/PeopleSlice/PeopleType.tsx

```diff
@@ -14,4 +14,9 @@ export interface Peoples {
 export interface PeoplesList {
   list: Peoples[];
   isLoading: boolean;
+  page: number;
+  total: number;
+  nextPageUrl: null | string;
```

Comment:
Please explain me the logic behind this ?

Reply:
I removed this issue